cmd/xl-v1-object.go

```diff
@@ -198,8 +198,14 @@ func (xl xlObjects) GetObject(bucket, object string, startOffset int64, length i
 		return traceError(InvalidRange{startOffset, length, xlMeta.Stat.Size})
 	}
 
+	// Calculate endOffset according to length
```

Comment:
We can return if 'offset == 0 && length == 0' as success that's it at object-handlers.go as an additional optimization so that the i/o doesn't even reach the backend as its not needed anyways.